scripts/scr_marine_struct/scr_marine_struct.gml

```diff
@@ -0,0 +1,386 @@
+// Script assets have changed for v2.3.0 see
+// https://help.yoyogames.com/hc/en-us/articles/360005277377 for more information
+
+
+global.trait_list = {
+	"shitty_luck":{
+		luck:-4
+	},
+	"very_hard_to_kill":{
+		constitution:10,
+		luck:2,
+	},
+	"paragon":{
+		constitution:6,
+		luck:2,
+		strength:2,
+		dexterity:2,
+		intelligence:2,
+		charisma:2
+	},
+	"warp_touched":{
+		intelligence:4
+	}
+}
+global.base_stats = { //tempory stats subject to change by anyone that wishes to try their luck
```

Comment:
hmm, I'd rename this to global.roles or global.titles or global.marine_titles or global.marine_roles

Reply:
it is neither roles, titles or marine roles these are simply pre set base stats for units of all kinds, that then have traits and other elements layered on top to create a given units full stats

Reply:
ok, seems fair